src/Workspaces/MSBuildTest/DotNetSdkMSBuildInstalled.LooseAssemblyVersionLoader.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+#if NETCOREAPP
+using System.Runtime.Loader;
+#endif
+
+namespace Microsoft.CodeAnalysis.MSBuild.UnitTests
+{
+    internal partial class DotNetSdkMSBuildInstalled
+    {
+#if NETCOREAPP
+
+        private static class LooseVersionAssemblyLoader
```

Comment:
This is taken from dotnet-format for resolving msbuild dependencies.

Reply:
Thanks for pushing back on this. This file had been necessary in dotnet-format since the beginning, but it looks like MSBuildLocator was updated in https://github.com/microsoft/MSBuildLocator/pull/107 to load assemblies from the MSBuild folder regardless of version. Will remove.